crates/wdk-build/src/cargo_make.rs

```diff
@@ -618,10 +618,17 @@ fn get_path_from_env(
 ) -> PathBuf {
     env::var(env_var).map_or_else(
         |e| {
-            trace!(
-                "Could not read env var '{env_var}': {e:?}. Constructing path from WDK content \
-                 root"
-            );
+            const FALLBACK_MSG: &str = "Constructing path from WDK content root instead";
+            match e {
+                env::VarError::NotPresent => {
+                    trace!("Env var '{env_var}' not found. {FALLBACK_MSG}")
+                }
+                env::VarError::NotUnicode(val) => {
+                    warn!(
+                        "Env var '{env_var}' contains non-UTF8 characters: {val:?}. {FALLBACK_MSG}"
```

Comment:
I think the message should say `non-unicode` instead of `non-UTF8`.

Reply:
Okay, what we need as a return value of this `get_path_from_env` function is a UTF8 string. 

So, the message could either mention that the environment variable contains a non-unicode value (not UTF8 or UTF16), or it could explain that because it has non-unicode content, it can't be a valid UTF8 string, which is required as our return value. From your explanation, it looks like we're choosing the second approach.